### PR TITLE
Add the tests used to bring the System V struct passing/returning fun…

### PR DESCRIPTION
**Pull request by LLITCHEV**
…ctionality.

There are:
1. Tests for passing managed struct.\
2. Tests for returning managed structs.
3. Tests for PInvoke and reverse PInvoke using callbacks and native SO.

### CODE REVIEW

**Comment by LLITCHEV** [comment]
@adityamandaleeka @janvorli @pgavlin Could you please take a look.

File: tests/src/JIT/Methodical/structs/systemvbringup/structpinvoketests.cs [hunk 1]
```diff
@@ -0,0 +1,1021 @@
...
+
+        // struct that doesn't fit into registers
+        public struct S20
+        {
+            public long x;
+            public long y;
+            public long z;
+            public long w;
+        }
+/* These tests are not working on non Windows CoreCLR
```
Comment (pgavlin) [inline tests/src/JIT/Methodical/structs/systemvbringup/structpinvoketests.cs]: Is there a bug tracking this?

Comment (LLITCHEV) [inline tests/src/JIT/Methodical/structs/systemvbringup/structpinvoketests.cs]: It is a known problem that is on the list for interop team to solve – feel free to ask @yizhang82  about the details.

Comment (LLITCHEV) [inline tests/src/JIT/Methodical/structs/systemvbringup/structpinvoketests.cs]: New issue #2076 added to track this.

**Comment by LLITCHEV** [comment]
Thanks!

**Pull request merged**